docs/blog/2019-04-05-health-brand-chooses-gatsby-to-level-up-its-modern-stack/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Health Brand Chooses Gatsby to Level-Up Its Modern Stack"
+date: 2019-04-05
+date: 2019-04-05
+author: Linda Watkins
+excerpt: "See how health brand Ritual used Gatsby to improve their SEO, page load times, delivery pipeline, and more!"
+tags:
+ - ritual
+ - react
+ - graphql
+---
+
+[Ritual](https://ritual.com/) is a health brand led by the belief that women should know exactly what’s in their vitamins. Ritual’s vitamins work smarter to fill the gaps in women’s diets with high quality nutrients. They’re on a mission to inspire women to feel their best, every day. Ritual’s clean, modern stack website, leads customers speedily through an educational purchase experience to learn more about the products, the ingredients, and overall benefits they can find by investing in their vitamin habits every day.
+
+We reached out to their Director of Engineering, Daniel Archer, to share ”Why Gatsby?” for their recent website rebuild. Here’s some of our conversation:
+
+## The Gatsby answer: Better code, SEO, page load times, and integration
+
+Ritual chose a rebuild with Gatsby due to the restrictions they were running into with liquid templates and UI-based content changes. While static and fast, the issues were making it more difficult to bring code up to their modern engineering standards. They also needed to enable other teams to quickly and easily change copy, imagery and new concepts. Both of these led them to look for a new solution - Gatsby.
+
+“We were looking for opportunities to combine a powerful CMS with React and Styled Components for maximum composability, while still maintaining a static-site for optimized SEO and page load times,” shared Daniel.
+
+In addition, they had some specific business and impact goals they were trying to achieve with the new site and top on that list was improving their development velocity.
+
+<pullquote citation="Daniel Archer">
+“We felt that investing in the combination of React, GraphQL, and a headless CMS as early as possible would allow us to streamline our delivery pipeline and iterate faster.”
+</pullquote>
+
+## Gatsby and React: Shorter builds, rapid changes
+
+Although they considered alternatives like Django, Ghost and Jekyll, they realized these other solutions would require significant new tooling, a longer timeline and a longer ramp-up period.
+
+“We first chose Gatsby because of the appeal with one popular Gatsby solution of using Contentful and Netlify,” Daniel shared. “We felt like it was a compelling ideal to empower our content and creative teams to make rapid changes to our site, and to enable our engineering team to develop in simple, reusable React components that work with a GraphQL interface.”
```

Comment:
```suggestion
<pullquote citation="Daniel Archer">
We first chose Gatsby because of the appeal with one popular Gatsby solution of using Contentful and Netlify. We felt like it was a compelling ideal to empower our content and creative teams to make rapid changes to our site, and to enable our engineering team to develop in simple, reusable React components that work with a GraphQL interface.
</pullquote>
```

Reply:
Added.